src/material/schematics/ng-generate/navigation/schema.json

```diff
@@ -92,6 +92,11 @@
       "type": "boolean",
       "default": false,
       "description": "Specifies if the component is an entry component of declaring module."
+    },
+    "routing": {
+      "type": "boolean",
+      "default": false,
+      "description": "Specifies whether Angular routing is enabled."
```

Comment:
I would add just a tiny bit more to this description:
"This controls whether anchor elements use `href` or `routerLink`"

Reply:
Done @jelbourn .